android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraModule.java

```diff
@@ -342,4 +610,15 @@ private File getTempMediaFile(int type) {
             return null;
         }
     }
+
+    private void addToMediaStore(Uri uri) {
+        if (uri == null) {
+            Log.e(TAG, "Tried to store null url to media store.");
+            return;
+        }
+
+        Intent mediaScannerIntent = new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE);
+        mediaScannerIntent.setData(uri);
+        _reactContext.sendBroadcast(mediaScannerIntent);
```

Comment:
Should use MediaScannerConnection instead.

https://developer.android.com/reference/android/media/MediaScannerConnection.html